.github/workflows/ci.yaml

```diff
@@ -342,6 +342,9 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - build_docker_images
+    strategy:
+      matrix:
+        test-target: [test-functions-k8s-undockerized, test-k8s-undockerized]
```

Comment:
When using a matrix strategy, the entire job runs multiple times.
Add the `test-target` value (or a human-readable version of it) to the name of the job so it will be easy to differentiate between the jobs